portfolio/src/main/webapp/style.css

```diff
@@ -85,33 +104,29 @@
   font-size: 20px;
 }
 
-.gallery:hover {
+div.gallery:hover {
```

Comment:
Why do you add "div" back? I'm not sure whether you need it. I deleted them for my codes and everything works fine.